src/state/content-model/types.js

```diff
@@ -1,2 +1,3 @@
 // eslint-disable-next-line import/prefer-default-export
```

Comment:
this comment is no longer necessary since there are two exports

Reply:
done